### PR TITLE
Specified tenant creation errors

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: test/api/TenantTest.ts [hunk 1]
```diff
@@ -268,7 +268,7 @@ describe('Tenant tests', function() {
       response = await CentralServerService.defaultInstance.createEntity(
         CentralServerService.defaultInstance.tenantApi, tenant, false);
       // Check
-      expect(response.status).to.equal(597);
+      expect(response.status).to.equal(598);
```
Comment (LucasBrazi06) [inline test/api/TenantTest.ts:271]: Use the constant.

File: src/server/rest/v1/service/TenantService.ts [hunk 2]
```diff
@@ -236,7 +236,7 @@ export default class TenantService {
     if (foundTenant) {
```
Comment (LucasBrazi06) [inline src/server/rest/v1/service/TenantService.ts:236]: I just saw that and to be honest we can remove this check.
There is no unicity on the tenant's name.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: test/api/TenantTest.ts (hunk 1 above)
Comment (Ninjeneer) [reply test/api/TenantTest.ts]: Done

File: src/server/rest/v1/service/TenantService.ts (hunk 2 above)
Comment (Ninjeneer) [reply src/server/rest/v1/service/TenantService.ts]: Ok, removed

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**